### PR TITLE
fix(multiedit): Fixed saving media on multiedit view

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
[SDESK-2366]

### CODE REVIEW

File: scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts [hunk 1]
```diff
@@ -324,9 +324,13 @@ export function ArticleEditDirective(
...
                             resetFieldMetadata(scope.item);
-                            scope.refresh();
 
-                            if (isPublished(scope.item)) {
+                            // On multiedit mode, there is no refresh function
+                            if (typeof scope.refresh === 'function') {
+                                scope.refresh();
+                            }
+
+                            if (isPublished(scope.item) || scope.articleEdit) {
```
Comment (tomaskikutis) [inline scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts]: What are you checking with ` scope.articleEdit`? I don't see such a variable in the directive.

File: scripts/core/editor3/helpers/fieldsMeta.ts [hunk 2]
```diff
@@ -50,7 +50,9 @@ export function setFieldMetadata(item, fieldKey, contentKey, contentValue) {
 }
 
 export function resetFieldMetadata(item) {
-    Object.keys(item[META_FIELD_NAME]).forEach((key) => {
-        setFieldMetadata(item, key, fieldsMetaKeys.draftjsState, null);
-    });
+    if (item[META_FIELD_NAME]) {
```
Comment (tomaskikutis) [inline scripts/core/editor3/helpers/fieldsMeta.ts]: When does it need this condition? Is it `null` sometimes? When?

File: scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts (hunk 1 above)
Comment (pavlovicnemanja) [reply scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts]: If scope.articleEdit is defined then it means it's called from Multiedit directive, which doesn't have item and originItem in scope, so autosave is not working.

File: scripts/core/editor3/helpers/fieldsMeta.ts (hunk 2 above)
Comment (pavlovicnemanja) [reply scripts/core/editor3/helpers/fieldsMeta.ts]: When this is triggered from multiedit it's failing:
https://github.com/superdesk/superdesk-client-core/pull/3179/files#diff-1718c3ee2990cd8f6be76101964abc0fR326

File: scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts (hunk 1 above)
Comment (tomaskikutis) [reply scripts/apps/authoring/authoring/directives/ArticleEditDirective.ts]: Makes sense. Can you add TypeScript types for `scope` and add a comment in the interface that `articleEdit` means that the item is being edited in multi-edit mode?

**Pull request merged**